packages/core/src/search/search.ts

```diff
@@ -146,8 +147,13 @@ export function parseSearchRequest<T extends Resource = Resource>(
  * @returns A parsed SearchRequest.
  */
 export function parseSearchUrl<T extends Resource = Resource>(url: URL): SearchRequest<T> {
-  const resourceType = url.pathname.split('/').filter(Boolean).pop() as ResourceType;
```

Comment:
Did this logic change?  I think it's the same?

Reply:
It should be the same: I was just trying to eliminate the throwaway array from `.filter().pop()`